tests/unit/resources/test_model.py

```diff
@@ -182,7 +182,7 @@ def test_resource_references(self):
         self.assertEqual(len(model.references), 1)
 
         ref = model.references[0]
-        self.assertEqual(ref.name, 'Frob')
+        self.assertEqual(ref.name, 'frob')
```

Comment:
This was actually a bug. This is a **reference**, and as such should have a snake-cased name.